server/main.lua

```diff
@@ -78,19 +78,19 @@ RegisterNetEvent('qb-cityhall:server:requestId', function(item, hall)
         return TriggerClientEvent('QBCore:Notify', src, ('You don\'t have enough money on you, you need $%s cash'):format(itemInfo.cost), 'error')
     end
     local metadata = {}
-    if item == "id_card" then
+    if item.item == "id_card" then
```

Comment:
Change this and the rest of the `item` references to `itemInfo.item` for it to work as you want

Reply:
Whoops, this was a mistake, got to setup that qbox only server